### ISSUE TITLE
BUG Fix parameter name in forest initialization

### ISSUE THREAD

**Issue by PSSF23** [neurodata/honest-forests#7]
@rflperry Observed in #6 

```python
TypeError: ForestClassifier.__init__() missing 1 required positional argument: 'estimator'
```

**Comment by sampan501** [comment]
I think the solution to this may be that the code initializes with `base_estimator` instead of `estimator`. May be a recent scikit-learn change

**Comment by PSSF23** [comment]
Ye according to their current documentation:

    estimator_ : :class:`~sklearn.tree.DecisionTreeClassifier`
        The child estimator template used to create the collection of fitted
        sub-estimators.
        .. versionadded:: 1.2
           `base_estimator_` was renamed to `estimator_`.
    base_estimator_ : DecisionTreeClassifier
        The child estimator template used to create the collection of fitted
        sub-estimators.
        .. deprecated:: 1.2
            `base_estimator_` is deprecated and will be removed in 1.4.
            Use `estimator_` instead.

I'll fix this as well in #6
